src/olympia/abuse/templates/abuse/emails/reporter_appeal_ignore.txt

```diff
@@ -0,0 +1,7 @@
+{% extends "abuse/emails/base_reporter.txt" %}{% load i18n %}{% block content %}{# L10n: This is an email. Whitespace matters #}{% blocktranslate %}
+Thank you for your report about {{ name }} on Mozilla Add-ons, and for providing more information about your concerns.
+
+After reviewing your appeal, we determined that the previous decision, that this content does not violate Mozilla’s policies (https://extensionworkshop.com/documentation/publish/add-on-policies/), was correct. Based on that determination, we have denied your appeal, and will not take any action against the account or the content.
```

Comment:
At least for translated emails, I think we want the link to the policies to be a variable passed through context. It might change when we clarify polices pages (especially as right now the link we have is somewhat add-on specific) and we don't want to re-translate the string when that happens

Reply:
added it for all the templates for completeness https://github.com/mozilla/addons-server/pull/21755/commits/373c96d3e054ee954eee0b4a3a78c272acf2a7f7